test/acceptance/features/volumeMount.feature

```diff
@@ -0,0 +1,362 @@
+Feature: Bindings get injected as files in application
```

Comment:
`volumeMount` is just technical details how we inject files, but the feature itself is about injecting binding data into application pods using files. Hence, let's rename this feature file to something like `injectBindingsAsFiles.feature`